src/qz/printer/info/NativePrinter.java

```diff
@@ -152,22 +151,16 @@ public void setResolution(PrinterResolution resolution) {
         return resolution;
     }
 
-    public List<Integer> getResolutions() {
+    public List<PrinterResolution> getResolutions() {
         // TODO: Test/Implement supported resolutions for CUPS
```

Comment:
Unfortunately, due to this `TODO:` item and lack of density reporting from CUPS, we can't go higher than what's reported in the `.ppd` file as `*DefaultResolution: 300dpi` on my Dymo LabelWriter 450 Turbo on Mac.

The Dymo `.ppd` comes back with the following when parsing for `dpi`:

```diff
+ *DefaultResolution: 300dpi
  *Resolution 300dpi/300 DPI: "<</HWResolution[300 300]>>setpagedevice"
  *Resolution 300x600dpi/300x600 DPI: "<</HWResolution[300 600]>>setpagedevice"
  *UIConstraints: *DymoPrintQuality Text       *Resolution       300x600dpi
  *UIConstraints: *DymoPrintQuality Graphics   *Resolution       300dpi
  *UIConstraints: *Resolution       300x600dpi *DymoPrintQuality Text 
  *UIConstraints: *Resolution       300dpi     *DymoPrintQuality Graphics
```

I wonder if we should include better multi-res parsing as part of this PR.  What do you think @bberenz?  If so, do you have some printers to scrape info from?  @klabarge if you can offer some info too, that would be great.

Reply:
The command is:

```bash
cat "$(lpstat -l -p |grep DYMO|grep ppd|cut -d' ' -f2)" |grep dpi
```

Just replace `DYMO` with a unique printer name.

Reply:
My Zebra comes back with the following:

```diff
+ *DefaultResolution: 203dpi
  *Resolution 203dpi/203dpi: "<</HWResolution[203 203]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
  *Resolution 300dpi/300dpi: "<</HWResolution[300 300]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
  *Resolution 600dpi/600dpi: "<</HWResolution[600 600]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
```

Reply:
My Epson WF4740 Workforce Pro shows the following (only returned one line so grep'd for `600` instead:

```diff
+ *DefaultResolution: 600x600dpi
  *cupsPrintQuality Normal: "<</cupsInteger1 4 /HWResolution[600 600]>>setpagedevice"
  *cupsPrintQuality High: "<</cupsInteger1 5 /HWResolution[600 600]>>setpagedevice"
```

Reply:
> `cat "$(lpstat -l -p |grep DYMO|grep ppd|cut -d' ' -f2)" |grep dpi`

Some of the `ppd` files on my system didn't have `dpi`. I had better luck `grepping` for "Resolution".

#### Epson WF 3730
```diff
+ *DefaultResolution: 600x600dpi
*cupsPrintQuality Normal: "<</cupsInteger1 4 /HWResolution[600 600]>>setpagedevice"
*cupsPrintQuality High: "<</cupsInteger1 5 /HWResolution[600 600]>>setpagedevice"
```

#### HP LaserJet 500 Color MFP M570
This printer didn't have a `DefaultResolution` nor anything with `dpi`

```diff
/HWResolution [600 600] /PreRenderingEnhance false >> setpagedevice"
/HWResolution [600 600] /PreRenderingEnhance false >> setpagedevice"
```

#### HP LaserJet M603
This printer didn't have a `DefaultResolution`
```diff
*HPPrintQuality FastRes1200/FastRes 1200: "<</HWResolution [600 600] /PreRenderingEnhance true>> setpagedevice"
+ *HPPrintQuality 600dpi/600 dpi: "<</HWResolution [600 600] /PreRenderingEnhance false>> setpagedevice"
*HPPrintQuality ProRes1200/ProRes 1200: "<</HWResolution [1200 1200] /PreRenderingEnhance false>> setpagedevice"
```

#### Zebra LP2844
```diff
+ *DefaultResolution: 203dpi
*Resolution 203dpi/203dpi: "<</HWResolution[203 203]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
*Resolution 300dpi/300dpi: "<</HWResolution[300 300]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
*Resolution 600dpi/600dpi: "<</HWResolution[600 600]/cupsBitsPerColor 1/cupsRowCount 0/cupsRowFeed 0/cupsRowStep 0/cupsColorSpace 3>>setpagedevice"
```